.vsts-pipelines/steps/install-build-prereqs.yml

```diff
@@ -0,0 +1,4 @@
+steps:
```

Comment:
What was the reasoning on separating this into a template versus in-lining it?

Reply:
Leftover from my experimentation of trying to get a better conditional execution of the task.  Added it back inline.